test/unit/server/web/edit/index.js

```diff
@@ -228,7 +228,7 @@ lab.experiment('GET', function () {
     });
     server.inject('/setsession', function (res) {
       var header = res.headers['set-cookie'];
-      var cookie = header[0].match(/(?:[^\x00-\x20\(\)<>@\,;\:\\'\/\[\]\?\=\{\}\x7F]+)\s*=\s*(?:([^\x00-\x20\'\,\;\\\x7F]*))/);
+      var cookie = header[0].match(/(?:[^\x00-\x20\(\)<>@\,;\:\\'\/\[\]\?\=\{\}\x7F]+)\s*=\s*(?:([^\x00-\x20\'\,\;\\\x7F]*))/); // eslint-disable-line no-control-regex, no-useless-escape
```

Comment:
didn't have enough regex foo to make eslint happy so ignored individual lines. might be nice to move this in a more managable branch to a reusable module so it's not repeated so often